files/slave.dockerfile

```diff
@@ -2,21 +2,20 @@ FROM docker.io/stackbrew/centos:latest
 MAINTAINER Chris Evich <cevich@redhat.com>
 
 RUN yum update -y && \
-    yum install -y findutils epel-release https://rdoproject.org/repos/rdo-release.rpm && \
+    yum install -y epel-release https://rdoproject.org/repos/rdo-release.rpm && \
     # Keep every layer as clean (i.e. small) as possible
     yum clean all
 
-# N/B: Use Ansible 2.1.0 until 2.1.2 is available b/c of:
-#      https://github.com/ansible/ansible/issues/15915
-
```

Comment:
Reverted, we're going the other way now - rpm version of 2.2.1 to source version of 2.2.1 (which is newer)